core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3190,7 +3197,7 @@ private boolean syncMetadata(RpcContext rpcContext, LockedInodePath inodePath,
         loadMetadata = true;
       } else {
         SyncResult result =
-            syncInodeMetadata(rpcContext, inodePath, syncDescendantType);
+            syncInodeMetadata(rpcContext, inodePath, syncDescendantType, pathsToLoad);
```

Comment:
Can this `pathsToLoad` info be incorporated into `SyncResult`? I think the `loadMetadata` bit can be changed to a group of paths to load.